packages/js/src/workouts/tailwind-components/IndexingError.js

```diff
@@ -0,0 +1,108 @@
+import PropTypes from "prop-types";
+import Alert from "./alert";
+import { __ } from "@wordpress/i18n";
+import RequestError from "../../errors/RequestError";
+import styled from "styled-components";
+
+const ErrorDetails = styled.div`
+	margin-top: 8px;
+`;
+
+const Preformatted = styled.pre`
+	overflow-x: scroll;
+	max-width: 500px;
+	border: 1px solid;
+	padding: 16px;
+`;
```

Comment:
We should probably kill styled components and instead use `<div>` and `<pre>` with the matching tailwind classnames.